README.md

```diff
@@ -562,11 +562,13 @@ In `Connect` tab, we'll configure Sonarr to send notifications to Plex when a ne
 
 #### Give it a try
 
-Let's add a serie !
+Let's add a series !
 
 ![Adding a serie](img/sonarr_add.png)
 
-Enter the serie name, then you can choose a few things:
+*Note: You may need to `chown -R $USER:$USER /path/to/root/directory` so Sonarr and the rest of the apps have the proper permissions to modify and move around files*
```

Comment:
Nit-picking: maybe add a final dot `.` to this line?

Reply:
Sure